.github/workflows/test.yml

```diff
@@ -1,11 +1,23 @@
 ---
```

Comment:
I saw that we did not change validate and unit-tests to use `ci/validate` and `ci/test-coverage`, was it on purpose or we just missed that?

Reply:
good catch, I've forgot to add ci/ prefix, we already have the targets.